tiledb/sm/query/query.cc

```diff
@@ -83,6 +83,13 @@ static uint64_t get_effective_memory_budget(
 /*   CONSTRUCTORS & DESTRUCTORS   */
 /* ****************************** */
 
+Query::CoordsInfo::CoordsInfo()
```

Comment:
I'm seeing a weird "possible use of uninitialized value" error in release builds, I changed this to try and rule something out.  No dice but having a constructor feels better.